SPECS/aws-nitro-enclaves-cli.spec

```diff
@@ -17,8 +17,8 @@
 
 Summary:    AWS Nitro Enclaves tools for managing enclaves
 Name:       aws-nitro-enclaves-cli
-Version:    1.0
-Release:    8%{?dist}
+Version:    1.1
```

Comment:
Version here should actually read 1.1.0 instead of 1.1; or actually 1.0.9 or something. The release number is a separate thing and it should not be related to the version number of the project itself.